### PR TITLE
workspace user friendly changes

### PR DESCRIPTION
**Pull request by Emyka**
A bunch of UX changes after the suggestion thread "User friendly updates to workspace UI" with the big help of Serena ❤️.

The following parts were moved from `Workspace` and `GuiHelpers` classes to their respective new classes:

- `BoneTree`
- `ControlButtons`: Gizmo Operation & extras buttons
- `AnimationControls`
- `PopupSelect`: HoverPopupWindow methods from GuiHelpers

### CODE REVIEW

File: Ktisis/Interface/Components/PopupSelect.cs [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+		private static int ColFromKey(int key) => key % HoverPopupWindowColumns;
+		private static int KeyFromRowCol(int row, int col) => (row * HoverPopupWindowColumns) + col;
+
+		public static void HoverPopupWindow(
+				HoverPopupWindowFlags flags,
+				IEnumerable<dynamic> enumerable,
+				Func<IEnumerable<dynamic>, string, IEnumerable<dynamic>> filter,
+				Action<dynamic> header,
+				Func<dynamic, bool, (bool, bool)> drawBeforeLine, // Parameters: dynamic item, bool isActive. Returns bool isSelected, bool Focus.
+				Action<dynamic> onSelect,
```
Comment (Fayti1703) [inline Ktisis/Interface/Components/PopupSelect.cs:52]: Probably shouldn't be using dynamic here and instead use generics (if possible)

File: Ktisis/Util/GuiHelpers.cs [hunk 2]
```diff
@@ -11,26 +11,30 @@
 
 using FFXIVClientStructs.Havok;
 
-using Ktisis.Structs.Bones;
-
 namespace Ktisis.Util
 {
 	internal class GuiHelpers {
-		public static bool IconButtonHoldConfirm(FontAwesomeIcon icon, string tooltip, bool isHoldingKey) {
+		public static bool IconButtonHoldConfirm(FontAwesomeIcon icon, string tooltip, bool isHoldingKey, Vector2 size = default) {
```
Comment (Fayti1703) [inline Ktisis/Util/GuiHelpers.cs:17]: This is going to result in a size of `(0, 0)`... is that intentional?

**Review by Fayti1703 — COMMENTED** [review]
In addition to the notes below: Commit cfef801 has...quite a long commit summary.
Try to keep your summaries short and move extra information into the rest of the commit message, separated by a blank line.

File: Ktisis/Util/GuiHelpers.cs (hunk 2 above)
Comment (Emyka) [reply Ktisis/Util/GuiHelpers.cs:17]: Yes, a size of `(0, 0)` means it's going to take the space it needs.

I wanted to keep the code clean without too much ifs and it seems this works as well as `(-1, -1)`, which was my first intention.

File: Ktisis/Interface/Components/PopupSelect.cs (hunk 1 above)
Comment (Emyka) [reply Ktisis/Interface/Components/PopupSelect.cs]: Very good suggestion!
Done in e10daa6.

**Comment by Emyka** [comment]
Sorry for the last minute change on f11b836, we had a discussion about it.
It is ready now.

**Pull request merged**